Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM node:16-bullseye AS builder
+
+WORKDIR /maci
+COPY . /maci/
+
+RUN npm i && \
+    npm run bootstrap && \
+    npm run build
```

Comment:
Build everything for now but eventually we'd build each package separately copying over the lerna generated artifacts and running `npx lerna link`